charts/incubator/libremdb/values.yaml

```diff
@@ -0,0 +1,36 @@
+image:
+  repository: kirk1984/libremdb
```

Comment:
Why that image?
From the gh repo, seems like it's mostly for personal use, and it can have a breaking change at any moment.

Reply:
The other one is 400mb.... we can switch it out though